src/components/SuggestedThings/Box.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import '../../../src/assets/css/Box.css';
+import suggestedThings from './suggestedThings';
+
+// This functions returns the individual suggested item image and name of a particular weather type.
+function Item(props) {
+    return (
+        <div key={props.name} className="flex suggested-thing">
+            <img className="icons" src={props.img} alt={props.alt} width="50" height="50"></img>
+            <h3>{props.name}</h3>
+        </div>
+    );
+}
+
+//This function returns the container which holds the three suggested items of a particular weather type.
+function Box(props){
+    const weatherType = props.weather.weather[0].main
+    return (
+        <div className="flex suggested-things-box">
+            {Object.keys(suggestedThings[weatherType]).map((item) => {
+                return (
+                    <Item
+                    img={suggestedThings[weatherType][item]}
+                    alt="Suggested thing images"
```

Comment:
1. `suggestedThings[weatherType]` can be taken out as a variable since it's being used at 2 places.
2. a better `alt tag` would be to show the `key` or maybe the `value` of dictionary/object of the suggested things

Reply:
I have made the necessary changes